operations/helm/charts/alloy/templates/controllers/daemonset.yaml

```
@@ -22,4 +22,7 @@ spec:
   updateStrategy:
     {{- toYaml . | nindent 4 }}
   {{- end }}
+  {{- with .Values.controller.terminationGracePeriodSeconds }}
+  terminationGracePeriodSeconds: {{ . }}
+  {{- end }}
```

```suggestion
  {{- if .Values.controller.terminationGracePeriodSeconds }}
  terminationGracePeriodSeconds: {{ .Values.controller.terminationGracePeriodSeconds | int }}
  {{- end }}
```
Force to an integer.